subliminal/subtitle.py

```diff
@@ -276,13 +275,16 @@ def guess_matches(video, guess, partial=False):
     matches = set()
     if isinstance(video, Episode):
         # series
-        if video.series and 'series' in guess and sanitized_string_equal(guess['series'], video.series):
+        if video.series and 'title' in guess and sanitized_string_equal(guess['title'], video.series):
             matches.add('series')
+        # series
+        if video.title and 'episode_title' in guess and sanitized_string_equal(guess['episode_title'], video.title):
+            matches.add('title')
```

Comment:
I prefered the series/title over title/episode_title. From an episode standpoint (which is what a video file is), the title is the title of the episode, not the title of the series.
I believe I already said that somewhere in guessit's bugtracker but both points are valid so it doens't really matter.